docs/bootstrap-sphinx.css_t

```diff
@@ -0,0 +1,302 @@
+/*
+ * bootstrap-sphinx.css
```

Comment:
I see it isn't atuo-generated but I still don't understand why it's necessary and why it is so long.

Reply:
To clarify , this bootstrap-sphinx.css_t file is not part of the auto-generated docs, and actually lives in the sphinx bootstrap theme folder (inside  C:\Python27\Lib\site-packages\sphinx_bootstrap_theme\bootstrap\static in my computer).
This file, along with the layout.html file, were edited to fix the sidebar issue. We can discuss options if we prefer to avoid editing these files.

Reply:
Thank you for clarifying @santiagogaray . It would be good to discuss this over a video call to make sure I understand what changes you made to the original bootstrap-sphinx.css_t and their impact on the final docs.  If you can think of a way to only code in the changes you want to make to the file without having to copy the entire file, that would be ideal.  Let's discuss more on the call.

Reply:
Completely agree!

Reply:
@santiagogaray, Thank you for this PR. If I understand correctly this file gets generated by the bootstrap plugin when the documents are being generated. So I assume even if we keep the edited version they will most likely be overwritten when we generate a new version of the docs. Or does sphinx use the old files if they are already there?

Reply:
See my comment here: https://github.com/ladybug-tools/ladybug-geometry/pull/49#discussion_r328649111

It should be done using the customization feature.

Reply:
Thanks @mostaphaRoudsari. Correct, this file is generated every time the sphinx plugin is run. So, yes, even if we edit the generated file, it will be reverted to the original in the next update.

Reply:
I think we can resolve this now that we agree that Santiago's solution is good.